README.md

```diff
@@ -1,122 +1,129 @@
+  more details
 - Create a new `pipeline` job (`New Item` -> Choose `Pipeline` option and enter job name `microprofile-test-suite`)
-- Copy content of Jenkinsfile into Pipeline script section and `Save`
-    - Change `linux` label in `agent { label 'linux' }` to contain label assigned to your slaves
-    - Change `java-1.8.0-oracle` in `jdk 'java-1.8.0-oracle'` to installed JDK label
-- Trigger build by `Build Now` button - NOTE FIRST RUN WILL FAIL DUE TO ISSUES <https://issues.jenkins-ci.org/browse/JENKINS-40574/> <https://issues.jenkins-ci.org/browse/JENKINS-41929> but all the subsequent runs will work
+- Copy content of [Jenkinsfile](./Jenkinsfile) into Pipeline script section and `Save`
+    + Change `linux` label in `agent { label 'linux' }` to reflect the label assigned to your Jenkins nodes
+    + Change `java-1.8.0-oracle` in `jdk 'java-1.8.0-oracle'` to a lable matching an installed JDK
+- Trigger the build by `Build Now` button - 
+  NOTE FIRST RUN WILL FAIL DUE TO ISSUES <https://issues.jenkins-ci.org/browse/JENKINS-40574/> 
+  and <https://issues.jenkins-ci.org/browse/JENKINS-41929> 
+  but all the subsequent runs will work
 
 ## Quick compilation of the code
-Can be used to ensure code changes are compilable, `-Djboss.dist=foo` is workaround to skip unpacking of WildFly zip.
+The following command can be used to ensure that code changes are compilable, `-Djboss.dist=foo` is just a workaround 
+to skip unpacking the WildFly zip.
 ```
 mvn clean verify -DskipTests -DskipITs -Djboss.dist=foo
 
 ./mvnw clean verify -DskipTests -DskipITs -Djboss.dist=foo
 ```
 
-## Target distributions and MicroProfile specs
-
-The default Maven behavior when running the above mentioned commands will be to 
-execute the test suite with its default settings - 
-i.e. being about MicroProfile, this resolves to the assumption that:
-
-* all of the MicroProfile specs modules test cases should be executed
-* if a server instance must be started up then the `standalone-microprofile.xml`
- configuration should be used.
-
-What described above would fit in the scenarios of Wildfly (which includes all 
-of the MP specs since v. 19.0.0.Final) and EAP XP server distributions.
-
-When it comes to the standard EAP stream bits tests, then the test suite 
-behavior should be different. 
-Specifically, only the test cases belonging to the modules which relate to 
-those MicroProfile specs which constitute the so called `Observability layer`
-should be executed and the `standalone.xml` file should be used when starting 
-the server instance up.
-
-The MicroProfile `Observability layer` specs include:
-* MicroProfile Config
-* MicroProfile Metrics
-* Microprofile Health
-* MicroProfile OpenTracing (no tests in here for this one)
-
-In order to test the _just_ this reduced set of MicroProfile specs, then, an 
-additional property should be provided to the command lines described earlier, 
-namely `-Dmp.specs.scope=observability`.
```

Comment:
Should we remove observability from pom files as well? Or at least create an issue to do it later as a followup task?

Reply:
Exactly, #249 logged.